kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/ReplaceQuestions.vue

```diff
@@ -2,6 +2,9 @@
 
   <div>
     <h1>Replace questions</h1>
+    <p v-for="(question,index) in quizForge.selectedActiveQuestions.value" :key="`${index}q`">
+      {{ JSON.stringify(question) }}
```

Comment:
not blocking: the `JSON.stringify(question)` here feels very weird. is there something about the data structure that requires this, whereas in other places we don't need it? Is it permanent or is this because of the debugging setup? I know that we do use JSON.stringify in places but I am struggling to think of other instances where we use it in a vue template.

Reply:
This will be wiped out in #11497 -- I just put it there as an example of like... "look here is where/how you get the questions"